gb-backend/src/main/groovy/nl/thehyve/gb/backend/client/TransmartRestClient.groovy

```diff
@@ -49,6 +49,7 @@ class TransmartRestClient extends AbstractRestClient {
     }
 
     private URI createURI(String urlParts) {
-        URI.create("$transmartServerUrl/$transmartApiVersion/$urlParts")
+        String validUrlParts = urlParts.replaceAll("\\s","%20")
```

Comment:
Maybe use [URLEncoder](https://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html) instead to cover more exotic cases altogather?

Reply:
URLEncoder was also my first idea, but it converts white spaces  to `+` character instead of `%20`. I didn't find a nicer way of encoding this.